model_compression_toolkit/core/pytorch/reader/reader.py

```diff
@@ -13,19 +13,31 @@
 # limitations under the License.
 # ==============================================================================
 
-
+import torch
 import logging
 from typing import Callable, Dict
-
-import numpy as np
-import torch
-from torch.fx import symbolic_trace
 from torch.fx.passes.shape_prop import ShapeProp
+from torch.fx import Tracer, GraphModule
 
 from model_compression_toolkit.logger import Logger
 from model_compression_toolkit.core.common import Graph
 from model_compression_toolkit.core.pytorch.reader.graph_builders import edges_builder, nodes_builder
 from model_compression_toolkit.core.pytorch.utils import set_model
+from sony_custom_layers.pytorch import CustomLayer
```

Comment:
Make sure that sony_custom_layers is part of MCT requirement

Reply:
Please see my comment regarding the requirements, I don't understand why to make it mandatory